platform/pulpcore/app/tasks/publisher.py

```diff
@@ -3,11 +3,20 @@
 from celery import shared_task
 
 from pulpcore.app import models
+from pulpcore.app import serializers
 from pulpcore.tasking.services import storage
 from pulpcore.tasking.tasks import UserFacingTask
 
 
 @shared_task(base=UserFacingTask)
+def update(publisher_id, data, *args, **kwargs):
```

Comment:
Can `*args` and `*kwargs` be converted to named positional and keyword parameters. We should never use `*args, and **kwargs` unless we don't know what will be passed in. Since we control the signature and the arguments sent when the task is dispatched we know the arguments in this case.